datasets/ar/parliament/crawler.py

```diff
@@ -51,10 +49,14 @@ def crawl_person(context: Context, element) -> dict:
     person_data["term"] = _extract_text(element, ".//td[4]/text()")
```

Comment:
This crawler is a bit of a mess because it creates this `person_data` intermediate object instead of parsing straight into an entity under construction. Would you perhaps be up for getting it rid of `person_data`? It's half a re-write, but we end up with a more sane crawler.